docs/rep/node-agent/README.md

```diff
@@ -0,0 +1,93 @@
+# RIK - Riklet Architecture
+
+This document provides a technical reflexion about our component, the **riklet**. Our component and the [rik-proxy]("https://example.com") will be installed on each worker node of a rik cluster. We will describe this component in more details throughout this document.
+
+Below a diagram to illustrate how these components work together :
+
+![Global architecture](https://i.imgur.com/ZwLWFYV.png)
+
+The **riklet** is the primary _node agent_ that runs on each node.
+
+## Glossary
+
+- `Pod`: Execution unit in Rik. It should be able to execute docker containers for the first release.
+- `Node`: A cluster unit that executes a workload and contains only one `pod` for the first release.
+
+## Riklet
+
+### Objectives
+
+Riklet is responsible to interpret scheduling instructions and run workloads in a dedicated pod.
+
+Also, it should be able to send metrics about node and pods regularly to the scheduler.
+
+### Communication
+
+In order to communicate with the scheduler and potentially other rik components, **riklet** had to expose an API.
```

Comment:
No, `riklet` connects to an exposed grpc API from the scheduler.

Reply:
We were wrong indeed, this will be changed in the next commit.